boards/hail/src/main.rs

```diff
@@ -83,6 +83,10 @@ struct Hail {
     ipc: kernel::ipc::IPC,
     crc: &'static capsules::crc::Crc<'static, sam4l::crccu::Crccu<'static>>,
     dac: &'static capsules::dac::Dac<'static>,
+    analog_comparator: &'static capsules::analog_comparator::AnalogComparator<
+        'static,
+        sam4l::acifc::Acifc<'static>,
+    >,
```

Comment:
:) This is the opposite, please put the pin comments back and remove the code that isn't really connected to anything.

Reply:
Do you want me to remove the referneces in the struct/impl as well?